src/Compatibility/GenAPI/Microsoft.DotNet.GenAPI/CSharpFileBuilder.cs

```diff
@@ -91,6 +81,8 @@ public void WriteAssembly(IAssemblySymbol assemblySymbol)
             compilationUnit = compilationUnit.NormalizeWhitespace(eol: Environment.NewLine);
 
             Document document = project.AddDocument(assemblySymbol.Name, compilationUnit);
+            //TODO: this seems to introduce improperly scoped attributes in places (Like [GuidAttribute("...")] instead of [global::System.Runtime...etc.])
+            //This is being handled via extra usings in the "header" for now. Seems buggy.
```

Comment:
Seems like that could be a bug in the simplifier - we have found a few roslyn bugs like this.  Usually what I do is create a standalone repro - file a bug - eg: https://github.com/dotnet/roslyn/issues/67867

Reply:
Yeah, I'll dig into this at some point.